packages/components/src/InputPhoneNumber/InputMask.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { ReactElement, cloneElement, useState } from "react";
+import styles from "./InputMask.css";
+import { FormFieldProps } from "../FormField";
+
+export interface InputMaskProps {
+  /**
+   * Determines the masking pattern. By default, a `*` is used to be the value
+   * you're expecting the user would type.
+   */
```

Comment:
Is this a regex? A string we treat specially? Consider being more precise and specific about the pattern format. 
Also, could we add an example usage in this comment?

Reply:
Hmmm I am picking up what you're putting down.

I'll add an example since this needs to be paired with the pattern to actually understand what's up haha

```tsx
<InputMask pattern="(n*n=n)" delimiter="n" />
```

Typing `122` on the field would give you a value of `1*2=2`

Reply:
This looks great, thanks!